src/cplscheme/tests/serial-explicit-coupling-datainit.xml

```diff
@@ -1,21 +1,28 @@
-<?xml version="1.0"?>
-
+<?xml version="1.0" encoding="UTF-8" ?>
 <configuration>
-   <data:scalar name="Data0"/>
-   <data:vector name="Data1"/>
-   <data:scalar name="Data2"/>
-   <mesh name="Mesh">
-      <use-data name="Data0"/>
-      <use-data name="Data1"/>
-      <use-data name="Data2"/>
-   </mesh>
-   <m2n:sockets from="Participant0" to="Participant1"/>
-   <coupling-scheme:serial-explicit>
-      <participants first="Participant0" second="Participant1"/>
-      <time-window-size value="0.1" method="fixed"/>
-      <max-time-windows value="1"/>
-      <exchange data="Data0" mesh="Mesh" from="Participant1" to="Participant0"/>
-      <exchange data="Data1" mesh="Mesh" from="Participant1" to="Participant0" initialize="on"/>
-      <exchange data="Data2" mesh="Mesh" from="Participant0" to="Participant1"/>
-   </coupling-scheme:serial-explicit>
+  <data:scalar name="Data0" />
+
+  <data:vector name="Data1" />
+
+  <data:scalar name="Data2" />
```

Comment:
Could we get rid of the empty lines here? So, no new line between `data` tags?

Reply:
Good point. It currently groups on the full name of the tags, hence the 3 groups.